lib/gui/os/window-progress.js

```diff
@@ -37,13 +71,20 @@ exports.currentWindow = electron.remote.getCurrentWindow();
  * @description
  * Show progress inline in operating system task bar
  *
- * @param {Number} percentage - percentage
+ * @param {Number} state - flashing metadata
  *
  * @example
- * windowProgress.set(85);
+ * windowProgress.set({
+ *   type: 'write',
+ *   percentage: 55,
+ *   speed: 2049
+ * });
  */
-exports.set = (percentage) => {
-  exports.currentWindow.setProgressBar(utils.percentageToFloat(percentage));
+exports.set = (state) => {
+  const title = makeTitle(initialTitle, state);
```

Comment:
But I guess `makeTitle` will become (a tiny bit) more complex if it incorporates your suggestion to "I think that a good default that doesn't require any extra knowledge is to just display `85% Flashing`, without any dash." ?